src/main/java/org/waarp/common/database/ConnectionFactory.java

```diff
@@ -141,8 +133,10 @@ private ConnectionFactory(DbProperties properties, String server, String user, S
         ds.setUsername(this.user);
         ds.setPassword(this.password);
         ds.setDefaultAutoCommit(true);
-        ds.setDefaultReadOnly(true);
+        ds.setDefaultReadOnly(false);
         ds.setValidationQuery(this.properties.getValidationQuery());
+        ds.setMaxActive(1000);
+        ds.setMaxActive(10000);
```

Comment:
Why 2 calls for `setMaxActive()` ?

Plus, 10 000 seems ***very*** high ! postgresql, for example, often has a default of 100 concurrent connections...
What happens if the database refuses a new connection ? 

Is it possible to adjust this number with the database limit (`min(N, SERVER_LIMIT)`) ? (postgres returns it with `SHOW max_connections;` f.e.)

Reply:
You are right I will fix this.